### DISCUSSION TITLE
Silk.NET/blog/apr-2023/silk2170

### DISCUSSION THREAD

**Discussion by giscus** [dotnet/Silk.NET#1396] [Webpage Comments]
# Silk.NET/blog/apr-2023/silk2170

Are you tired of wading through messy code to achieve the smooth performance you need for your GPU applications? If so, it's time to upgrade to Silk.NET 2.17. This latest update delivers faster, cleaner code and a smoother experience than ever before, making it the easiest way to achieve top-tier performance without sacrificing usability. With Silk.NET 2.17, you can experience the power of clean code, achieve silky-smooth performance, and write code that feels right at home with the rest of your C# code.

https://dotnet.github.io/Silk.NET/blog/apr-2023/silk2170.html

**Comment by dibertz** [comment]
We kindly request that team Stride and Skill.Net support WASM so that we can export our video games made in Stride Engine to web format. 🙏

**Reply by Beyley** [reply]
Perksey wrote a [blog post](https://perksey.com/blog/blazor-silkdotnet.html) about why WASM support is not happening until after the 3.x series + the runtime matures more
the dotnet WASM support is just too immature right now

**Comment by XiaochaoYan** [comment]
Hi thanks for the great work. But I really have had to say that having documentation will really helping out a lot. ( especially for newbee like me
